src/react-native/13-making-http-requests/making-http-requests.md

```diff
@@ -87,11 +91,15 @@ It will be helpful to have a third terminal tab for the `npm run api` command.
 
 @sourceref ../../../exercises/react-native/13-http-requests/01-problem/src/components/Loading/Loading.test.tsx
 
-### Exercise
+✏️ Update **src/screens/StateList/StateList.test.tsx** to be:
+
+@diff ../../../exercises/react-native/12-navigation-params/01-solution/src/screens/StateList/StateList.test.tsx ../../../exercises/react-native/13-http-requests/01-problem/src/screens/StateList/StateList.test.tsx only
+
+### Exercise 1
 
 TODO
```

Comment:
is Exercise 1 supposed to be filled out with instructions or `TODO` still expected for this pr?

Reply:
Yep, creating excretes is in the content portion